libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc

```diff
@@ -136,7 +136,5 @@ Example:
 -------------------------------------------------------------------------------------
 `kube_config`:: (Optional) Use given config file as configuration for Kubernetes
 client. It defaults to `KUBECONFIG` environment variable if present.
-`default_indexers.enabled`:: (Optional) Enable/Disable default pod indexers, in
-case you want to specify your own.
-`default_matchers.enabled`:: (Optional) Enable/Disable default pod matchers, in
-case you want to specify your own.
+`default_indexers.enabled`:: (Optional) Enable or disable default pod indexers to specify your own.
```

Comment:
```suggestion
`default_indexers.enabled`:: (Optional) Enable or disable default pod indexers when you want to specify your own.
```

Reply:
my edit is wordier, but it makes it clear that you need to do more than enable or disable the default indexers.